<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Titr</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>حذف</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Titr</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>اضافه</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="meaningBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Book Antiqua'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Titr</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>جستجو</string>
    </property>
   </widget>
   <widget class="QLabel" name="guideLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Kamran</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>کلمه ای را وارد نمایید...</string>
    </property>
   </widget>
   <widget class="QLabel" name="loadLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>220</y>
      <width>63</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="suggestBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
